cmd/solvercli/cmd/relay.go

```diff
@@ -104,7 +104,7 @@ var relayCmd = &cobra.Command{
 			lmt.Logger(ctx).Error("Error creating hyperlane multi client from config", zap.Error(err))
 		}
 
-		destinationTxHash, destinationChainID, err := hyperlane.NewRelayer(hype, storageOverrideMap).Relay(ctx, originChainID, originTxHash, nil)
+		destinationTxHash, destinationChainID, _, err := hyperlane.NewRelayer(hype, storageOverrideMap).Relay(ctx, originChainID, originTxHash, nil)
```

Comment:
it could be helpful to add an optional maxTxFee flag to this command and set it here instead of passing nil in the last parameter

Reply:
agree, probably makes sense for another pr tho

Reply:
let's add this later if it becomes an issue